src/App.tsx

```diff
@@ -621,25 +621,55 @@ const Test = connect(
     })
 )(TestComponent as React.ComponentType<TestProps>);
 
-export default class App extends React.Component {
+interface AppState {
+    resetDialog: boolean;
+}
+
+export default class App extends React.Component<{}, AppState> {
+
+    constructor() {
+        super({});
+        this.state = { resetDialog: false };
+    }
+
     resetGame = () => {
```

Comment:
I think this is all already implemented. Won't this conflict?

Reply:
Works fine for me---I think this replaces the one you had.